src/observable/FromEventObservable.ts

```diff
@@ -34,13 +35,28 @@ function isEventTarget(sourceObj: any): sourceObj is EventTarget {
 
 export type EventTargetLike = EventTarget | NodeStyleEventEmmitter | JQueryStyleEventEmitter | NodeList | HTMLCollection;
 
+export type EventListenerOptions = {
+  capture?: boolean;
+  passive?: boolean;
+  once?: boolean;
+} | boolean;
+
+export type SelectorMetodSignature<T> = (...args: Array<any>) => T;
+
 /**
  * We need this JSDoc comment for affecting ESDoc.
  * @extends {Ignored}
  * @hide true
  */
 export class FromEventObservable<T, R> extends Observable<T> {
 
+  /* tslint:disable:max-line-length */
+  static create<T>(target: EventTargetLike, eventName: string): Observable<T>;
+  static create<T>(target: EventTargetLike, eventName: string, selector: SelectorMetodSignature<T>): Observable<T>;
+  static create<T>(target: EventTargetLike, eventName: string, options: EventListenerOptions): Observable<T>;
+  static create<T>(target: EventTargetLike, eventName: string, selector: SelectorMetodSignature<T>, options: EventListenerOptions): Observable<T>;
```

Comment:
@kwonoj, I have added overloaded signatures for `create` now. I see the point of having polymorphic methods here.

However, I find a strange behavior when calling the method:

```javascript
const options = { falseProp: 42 };
const subscription = Observable.fromEvent(window, 'scroll', options), // OK
const subscription = Observable.fromEvent(window, 'scroll', { falseProp: 42 }), // Typescript error
``` 
This feels like a typescript bug, you can see an example [here](http://www.typescriptlang.org/play/#src=type%20SelectorMetodSignature%20%3D%20number%3B%0D%0Atype%20EventListenerOptions%20%3D%20%7B%0D%0A%20%20capture%3F%3A%20boolean%3B%0D%0A%20%20passive%3F%3A%20boolean%3B%0D%0A%20%20once%3F%3A%20boolean%3B%0D%0A%7D%20%7C%20boolean%3B%0D%0A%0D%0Afunction%20create(eventName%3A%20string)%3A%20number%3B%0D%0Afunction%20create(eventName%3A%20string%2C%20selector%3A%20SelectorMetodSignature)%3A%20number%3B%0D%0Afunction%20create(eventName%3A%20string%2C%20options%3A%20EventListenerOptions)%3A%20number%3B%0D%0Afunction%20create(eventName%3A%20string%2C%20selector%3A%20SelectorMetodSignature%2C%20options%3A%20EventListenerOptions)%3A%20number%3B%0D%0A%0D%0A%0D%0Afunction%20create(eventName%3A%20string%2C%20selector%3F%3A%20SelectorMetodSignature%2C%20options%3F%3A%20EventListenerOptions)%20%7B%0D%0A%09return%201%3B%0D%0A%7D%0D%0A%0D%0Aconst%20options%20%3D%20%7Bcaptures%3A%20true%7D%3B%0D%0A%0D%0Acreate('click'%2C%20options)%3B%20%2F%2F%20OK%0D%0Acreate('click'%2C%20%7Bcaptures%3A%20true%7D)%3B%20%2F%2F%20Error%0D%0A).